lib/services/authority.go

```diff
@@ -408,6 +441,32 @@ func (c *CertAuthorityV2) GetTLSKeyPairs() []TLSKeyPair {
 	return c.Spec.TLSKeyPairs
 }
 
+// JWTKey return the first JWT keypair from the list of keypairs.
+func (c *CertAuthorityV2) JWTKey() (*jwt.Key, error) {
+	if len(c.Spec.JWTKeyPairs) == 0 {
+		return nil, trace.BadParameter("not JWT keypairs found")
+	}
+	key, err := jwt.New(&jwt.Config{
+		PublicKey:  c.Spec.JWTKeyPairs[0].PublicKey,
+		PrivateKey: c.Spec.JWTKeyPairs[0].PrivateKey,
+	})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return key, nil
+}
```

Comment:
how will the proxy verify both old and new tokens during rotation?
maybe we need an aggregate version of `jwt.Key`, which signs with one key but verifies with multiple?